src/lint/lint.ts

```diff
@@ -45,10 +45,9 @@ export async function lint(options: Options, config: ProjectConfig) {
   const rules = lintLib.registry.getRules(ruleCodes || lintOptions.rules);
   const filter = new WarningFilter({
     warningCodesToIgnore: new Set(lintOptions.ignoreWarnings || []),
-    minimumSeverity: Severity.WARNING
+    minimumSeverity: options.minSeverity || lintOptions.minSeverity || Severity.WARNING,
```

Comment:
Would still need to add `minSeverity` to the `polymer-project-config` `lint` section.

Side Note: wasn't sure if you do trailing commas in this repo or not.